.github/workflows/ci_vtests.yml

```diff
@@ -4,47 +4,239 @@ on:
   pull_request:
     branches:
     - master
+    - 4.0.2
 
 jobs:
   setup:
-    name: "VTests workflow"
+    name: "Setup VTests workflow"
     runs-on: ubuntu-20.04
+    outputs:
+      do_run: ${{ steps.output_data.outputs.do_run }}
+      reference_sha: ${{ steps.output_data.outputs.reference_sha }}
+      artifact_name: ${{ steps.output_data.outputs.artifact_name }}
     steps:
     - name: Cancel Previous Runs
-      uses: styfle/cancel-workflow-action@0.11.0
+      uses: styfle/cancel-workflow-action@0.9.1
```

Comment:
You're accidentally downgrading these actions. This one should be `0.11.0`, and the others should be `v3` everywhere instead of `v2`. Otherwise, we will get a bunch of warnings and those old ones will stop working at some point.

Reply:
Thanks for the notification, fixed